docs/auth.md

```diff
@@ -108,6 +108,12 @@ When the `Run` executes, before steps execute, a `~/.ssh/config` will be
 generated containing the key configured in the `Secret`. This key is then used
 to authenticate when retrieving any `PipelineResources`.
 
+Note: SSH credentials can also be used when invoking vanilla `git` commands
+in your Task's Steps. However, you will first need to symlink `/tekton/home/.ssh`
+to `/root/.ssh` because `ssh` ignores the `$HOME` environment variable, which by
```

Comment:
NIT: Perhaps instead of just `/root/.ssh` we could say to your user `.ssh` folder - e.g. `/root/.ssh` when using root.

Reply:
Good call, I've updated to include that, and added a little more detail about why this is necessary. Also turned this change into its own section because the content spans multiple paragraphs.